### ISSUE TITLE
Heroku Application Error

### ISSUE THREAD

**Issue by deejaywoody** [rauchg/slackin#309]
Hello,

I just setup Heroku and provided the necessary API info, etc. When I try to access, I'm seeing the following:

2017-05-25T12:34:59.443839+00:00 heroku[router]: at=error code=H10 desc="App crashed" method=GET path="/" host=mxeslack.herokuapp.com request_id=1691e64e-8b12-4621-9298-aba6804da673 fwd="74.138.35.21" dyno= connect= service= status=503 bytes= protocol=https

Based on what I read below, this stems from the account used not being an Admin. My question is: Can  Slackin/Heroku be used w. a free Slack team? I'm not finding anywhere to create a new @slackin-inviter account, so that's what keyed me into thinking this may not work with the free team.

Regards,

Eric

**Comment by lukcha** [comment]
I had the same Heroku error appearing at about the same time (6 days ago). We also use a free Slack team. I managed to get things working without issue by redeploying to Heroku as a fresh app, using the *legacy* Slack API keys and then re-adding the custom domain/subdomain in Heroku settings. Interesting to note the frontend was showing the same Heroku app error page until I re-added the custom subdomain. Surely that can't be where the issue stemmed from originally, unless the log issues were because of some domain path somehow not being passed through successfully, or something similar.

**Comment by deejaywoody** [comment]
Thanks for the information, @lukcha. Good to know that it will work with a free Slack team. Curious, where did you create the @Slackin-Inviter user inside the Slack API? For whatever reason, I have not seen an area to add a local user.

**Comment by lukcha** [comment]
Nope, I just use my own user account (which is more personal). In the past I've changed the address on my own account to suit the inviter, or just set up a separate admin account. Not sure that was the root cause of your problem tho.

**Comment by xuanzi23** [comment]
I have the "Application Error" after I deploy. Is it because my team is using free?
Must paid account only can use?

**Comment by KvalheimRacing** [comment]
@rauchg You guys should add in the readme that the Heroku slack domain adress _only requires the team specific adress_, not the entire slack adress. So where it stands `SLACK_SUBDOMAIN`, you should only write `myteamname`. You will get an H10 error code if you write `myteamname.slack.com`.

**Comment by deejaywoody** [comment]
@lukcha Finally got around to working on this. Legacy Slack API did the trick. Thanks for the info!
